docs/content/en/docs/config/grpc/services/storageprovider/_index.md

```diff
@@ -33,71 +33,47 @@ driver = "local"
 {{% /dir %}}
 
 {{% dir name="drivers" type="map[string]map[string]interface{}" default="docs/config/packages/storage/fs" %}}
-The configuration for the storage driver. [[Ref]](https://github.com/cs3org/reva/tree/master/internal/grpc/services/storageprovider/storageprovider.go#L55)
+ [[Ref]](https://github.com/cs3org/reva/tree/master/internal/grpc/services/storageprovider/storageprovider.go#L54)
```

Comment:
Also, blob permalinks can be considered instead of refs. to master. In case master changes those will keep the original line content referenced

```suggestion
 [[Ref]](https://github.com/cs3org/reva/blob/d405ec7b6c937f74df1480bc0f55d84c80d24bc4/internal/grpc/services/storageprovider/storageprovider.go#L54)
```

Reply:
@ishank011 , indeed, this could be a nice way Ishank to make sure the links always point to the contents in the PR

Reply:
Will look into it. Thanks!